publish.sbt

```diff
@@ -0,0 +1,8 @@
+publishTo := {
+    val corporateRepo = "http://toucan.simplesys.lan/"
+    if (isSnapshot.value)
+        Some("snapshots" at corporateRepo + "artifactory/libs-snapshot-local")
+    else
+        Some("releases" at corporateRepo + "artifactory/libs-release-local")
+}
+credentials += Credentials(Path.userHome / ".ivy2" / ".credentials")
```

Comment:
This is not mergable. SBT native packager is an open source project and does not contain
any cooperate build information.